### DISCUSSION TITLE
HTTP requests possible in server actions?

### DISCUSSION THREAD

**Discussion by VMLuca** [vercel/next.js#66114] [App Router] [answered]
We have a setup where we run our next.js application on Vercel, but the backend stuff (database, data handling) is on a different server. So we actually have 2 servers, one for delivering the UI (Vercel) and the other for data handling.
I did a bit of digging on server actions and all the examples I found were about using only one server where you can access the database directly.

So my question is: Is it ok to put a post request in the server actions to reach out to our second server?

**Comment by icyJoseph** [comment, marked as answer]
Yes you should be able to forward a request from the server action to a 3rd party, just remember that this would run in the context of the server, so cookies and other stuff that the client has set, won't be available when you craft the request from within the server action.

**Reply by VMLuca** [reply]
Thank you very much for the information!
